deeprankcore/features/contact.py

```diff
@@ -13,21 +13,28 @@
 
 MAX_COVALENT_DISTANCE = 2.1
 
-def _get_coulomb_potentials(atoms: List[Atom], distances: np.ndarray) -> np.ndarray:
+def _get_coulomb_potentials(atoms: List[Atom], distances: np.ndarray, cutoff_distance: float) -> np.ndarray:
     """ 
         Calculate Coulomb potentials between between all Atoms in atom.
         Warning: there's no distance cutoff here. The radius of influence is assumed to infinite.
             However, the potential tends to 0 at large distance.
     """
 
+    clamped_distances = distances
+    clamped_distances[distances == 0.0] = 3.0
```

Comment:
From where does this come from? (value of 3 for 0 distances)

Reply:
This came from:
https://github.com/DeepRank/deeprank/blob/3594e05dca15e688c10ef66e0fbc7cd71fd40027/deeprank/features/AtomicFeature.py#L617